CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2023-03-07
+
+* New ci-common-v3 for VFX Platform 2023 based on:
+  * C++17 (unchanged)
+  * Ccache 4.7.4 (from 4.0)
+  * Clang 14.0.6 (from 14.0.0)
```

Comment:
For 2023, can we upgrade to the current clang 15 as the default?

Reply:
A minor observation from me; this will also implicitly mean LLVM 15 which is a bit of a pain for us due to the opaque pointer type changes they made. So long as we continue to have 14 images though then I don't really have a problem with 15 as the default

Reply:
Should be possible to have clang14 and clang15 containers, I'll look into that. Looks like 15.0.0 is most recent clang version?

Reply:
llvm 15 supports opaque pointers, but does not require them. llvm 16 will shift entirely to opaque pointers. Thus, 15 is the one we need so that projects can start the transition, with both options available.

Reply:
Eh I tried toggling opaque pointers off with unsuccessful results. In any case, as long as 14 is an available option it's okay

Reply:
I did it just fine for OSL. I'll try to the find the PR to point you to.

Reply:
https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/pull/1592

Reply:
Thanks @lgritz, this was what I tried but I'll give it another whirl

Reply:
Next intermediate commit will have definitions for clang 14.0.6 and clang 15.0.7